checks/raw/license.go

```diff
@@ -38,6 +38,7 @@ const (
 	anyExt       = ".[^./]"
 	ofl          = "ofl"
 	patents      = "patents"
+	gpl          = "gpl(v|-)\\d"
```

Comment:
I'm confused - the unit tests seem to suggest we already handle the cases specified in #2159 (i.e apart from GPL no other change was added here). Then why did our cron job not catch these files?

Reply:
Yes, most things were already captured by the existing behavior, at least at the level of `checkLicense`.

Let me confirm with the cron results

Reply:
I misread the paper, thanks for pointing this out @spencerschrock